modules/openapi-generator/src/main/resources/cpp-restbed-server/api-header.mustache

```diff
@@ -76,6 +76,10 @@ private:
 		{{#allParams}}{{{dataType}}} const &{{#hasMore}}, {{/hasMore}}{{/allParams}}
 	)> handler_{{httpMethod}}_;
 	{{/vendorExtensions.x-codegen-otherMethods}}
+
+	{{#allParams}}
+	{{{dataType}}} {{paramName}};
```

Comment:
> You could initialize paramName with braces, so it would be `{{{dataType}}} {{paramName}}{};`

That would definitely work with all members that are not generated by a model. Other than that, definitely a good idea, I've just been folowing the style in the already existing cpp-restbed generator!

> This works for any type unless there are some special requirements when initializing it.

Precisely! Some models might contain self, cross or circular references. Which would initialize infinite objects when parsing the JSON. For that reason I've decided to initialize models as obejcts when actually required in `fromPropertyTree()`.